chatkit/build.gradle

```diff
@@ -17,15 +18,12 @@ android {
     }
 }
 
-ext {
-    supportVersion = '27.1.1'
-    flexboxVersion = '3.0.0'
-}
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(dir: "libs", include: ["*.jar"])
 
-    implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'com.google.android.material:material:1.0.0'
-    implementation "com.google.android.flexbox:flexbox:$flexboxVersion"
+    implementation "androidx.appcompat:appcompat:$appcompat_version"
+    implementation "com.google.android.material:material:$material_design_version"
+    implementation "com.google.android.flexbox:flexbox:$flexbox_version"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
New lib, why is this needed?

Reply:
Build error, missing a view inside Chatkit SDK. Adding lib fixed it. Think it was a dep previously in older version of flexbox.